book/src/validator-manager-move.md

```diff
@@ -0,0 +1,181 @@
+- `--http-port 5062`
+- `--enable-doppelganger-protection`
+
+Therefore, the destination VC command might look like:
+
+```bash
+lighthouse \
+    vc \
+    --http \
+    --unencrypted-http-transport \
+    --http-address 127.0.0.1 \
+    --http-port 5062 \
+    --enable-doppelganger-protection
+```
+
+> The `--enable-doppelganger-protection` flag is not *strictly* required, however
+> it is recommended for an additional layer of safety. It will result in 3-4
+> epochs of downtime for the validator after it is moved, which is generally an
+> inconsequential cost in lost rewards or penalties.
+> 
+> Optionally, users can add the `--http-store-passwords-in-secrets-dir` flag if they'd like to have
+> the import validator keystore passwords stored in separate files rather than in the
+> `valdiator-definitions.yml` file. If you don't know what this means, you can safely omit the flag.
+
+### 3. Obtain the Source API Token
+
+The VC API is protected by an *API token*. This is stored in a file on each of the hosts. Since
+we'll be running our command on the destination host, it will need to have the API token for the
+source host on its file-system.
+
+On the **source host**, find the location of the `api-token.txt` file and copy the contents. The
+location of the file varies, but it is located in the "validator directory" of your data directory,
+alongside validator keystores. For example: `~/.lighthouse/mainnet/validators/api-token.txt`.
+
+Copy the contents of that file into a new file on the **destination host** at `~/src-token.txt`. The
+API token should be similar to `api-token-0x03eace4c98e8f77477bb99efb74f9af10d800bd3318f92c33b719a4644254d4123`.
+
+### 4. Create an SSH Tunnel
+
+In one terminal window, SSH to the **destination host** and establish a reverse-SSH connection
+between the **desination host** and the **source host**.
+
+```bash
+ssh dest-host
+ssh -L 6062:localhost:5062 src-host
+```
+
+It's important that you leave this session open throughout the rest of this tutorial. If you close
+this terminal window then the connection between the destination and source host will be lost.
+
+### 5. Move
+
+With the SSH tunnel established between the `dest-host` and `src-host`, from the **destination
+host** run the command to move the validators:
+
+```bash
+lighthouse \
+	validator-manager \
+	move \
+	--src-vc-url http://localhost:6062 \
+	--src-vc-token ~/src-token.txt \
+	--dest-vc-url http://localhost:5062 \
+	--dest-vc-token ~/.lighthouse/mainnet/validators/api-token.txt \
+	--validators all
+```
+
+The command will provide information about the progress of the operation and
+emit `Done.` when the operation has completed successfully. For example:
+
+```bash
+Running validator manager for mainnet network
+Validator client is reachable at http://localhost:5062/ and reports 2 validators
+Validator client is reachable at http://localhost:6062/ and reports 0 validators
+Moved keystore 1 of 2
+Moved keystore 2 of 2
+Done.
+```
+
```

Comment:
Added more information on the VC log:
```suggestion
At the same time, `lighthouse vc` will log:
```bash
INFO Importing keystores via standard HTTP API, count: 1
INFO Enabled validator                       voting_pubkey: 0xab6e29f1b98fedfca878edce2b471f1b5ee58ee4c3bd216201f98254ef6f6eac40a53d74c8b7da54f51d3e85cacae92f, signing_method: local_keystore
INFO Modified key_cache saved successfully
```